src/defaults/ALI_my_defaults.m

```diff
@@ -0,0 +1,65 @@
+% (C) Copyright 2019 CPP BIDS SPM-pipeline developers
+
+function ALI_my_defaults()
+
+  %
+  % Short description of what the function does goes here.
+  %
+  % USAGE::
+  %
+  %   [argout1, argout2] = templateFunction(argin1, [argin2 == default,] [argin3])
+  %
+  %
+  %
+  % This is where we set the defautls we want to use for the ALE (lesion) toolbox.
+  % These will overide the spm defaults.
+  % When "not enough" information is specified in the batch files, SPM falls
+  % back on the defaults to fill in the blanks. This allows to make the
+  % script simpler.
+  
+  global defaults
```

Comment:
in general, for CPP SPM, we try to avoid declaring global variables because they are hard to track what is going on. 
Seach "why are global variables bad" if you want to drown in explanations but in short.

When you read code, you get an idea of what a function does first by seeing its declaration:

```
function opt = this_functioon(opt, arg2)
```

It makes sense to assume that this function modifies `opt` in some way.

But if `opt` is a global variable, then you can have:

```
function this_functioon(arg2)
```

And `opt` can still be modified and you would never know it: that's side-effect that is not obvious from the very top-level of the code.

So in this case what I would do is to return this `default` as an argument and then we will add it to the `opt` structure
that combines all the defaults we use for the pipeline.

We can modify `checkOptions` to do that. :-)